src/components/index/page-sections/ServiceSection.jsx

```diff
@@ -17,9 +17,9 @@ const ServiceSection = () => {
           link="/softwarelaboratory"
```

Comment:
sollten wir uns eine todo dran setzen. ich finde das nicht so meg schön. Lieber wäre mir glaube ich 3 componenten die spezifisch den inhalt halten / linkage definieren

Reply:
To Do bei Notion eingetragen.